src/pages/s/[[...all]].tsx

```diff
@@ -98,11 +98,13 @@ export const getServerSideProps: GetServerSideProps = async function ({
   const noIndexInitial = queryParamsPresent || noHits || userQueryPresent
 
   return {
-    props: {
-      resultsState: {rawResults},
-      initialSearchState,
-      pageTitle,
-      noIndexInitial,
-    },
+    props: JSON.parse(
```

Comment:
forcing JSON :/

Reply:
parsing json doesn't result in json, technically speaking, it results in a JS object.

another consideration here would be to stop destructuring the resultsState and then using the prop shorthand? vI'll pull the branch and have a peek.

Reply:
ah, I see, some weird ass deep object from Algolia doesn't play nice with Next. Cool lol